base/operators.jl

```diff
@@ -282,9 +282,10 @@ const ≠ = !=
 
 Determine whether `x` and `y` are identical, in the sense that no program could distinguish
 them. First the types of `x` and `y` are compared. If those are identical, mutable objects
-are compared by address in memory and immutable objects (such as numbers) are compared by
-contents at the bit level. This function is sometimes called "egal".
-It always returns a `Bool` value.
+are compared by address in memory, primitive objects (such as numbers) are compared by
```

Comment:
I know this is somewhat pre-existing, but giving "numbers" as an example is somewhat misleading (e.g. `Complex` or `BigFloat`).

Reply:
I think the latest revision addresses this.